### PR TITLE
add rint function for numpy frontend[2]

### PR DESCRIPTION
**Pull request by sagnik1511**
#5596

### CODE REVIEW

File: ivy/functional/frontends/numpy/mathematical_functions/rounding.py [hunk 1]
```diff
@@ -3,15 +3,11 @@
 
 # local
 from ivy.func_wrapper import from_zero_dim_arrays_to_float
-from ivy.functional.frontends.numpy.func_wrapper import (
-    to_ivy_arrays_and_back,
```
Comment (jkeane508) [inline ivy/functional/frontends/numpy/mathematical_functions/rounding.py:7]: Can I ask why you removed this?

**Comment by sagnik1511** [comment]
That was by mistake. Updating the section.

File: ivy/functional/frontends/numpy/mathematical_functions/rounding.py [hunk 2]
```diff
@@ -29,11 +25,29 @@ def ceil(
...
+    /,
+    out=None,
+    *,
+    where=True,
+    casting="same_kind",
+    order="k",
+    dtype=None,
+    subok=True,
+):
+    ret = ivy.rint(x, out=out)
```
[the remark this answers is not in the document]
Comment (sagnik1511) [reply ivy/functional/frontends/numpy/mathematical_functions/rounding.py:50]: Yes. I also found that. But I thought I only have to come up with the frontend part. Updating that too.

[the remark this answers is not in the document]
Comment (jkeane508) [reply ivy/functional/frontends/numpy/mathematical_functions/rounding.py:50]: Yep, if there isn't a direct implementation in the backend API, you need to create an implementation using a composition (combination) of Ivy backend functions. If this proves too difficult and we decide its better for Ivy to have its own implementation then we can add it to the extensions list for this frontend :)

**Pull request closed without merging**